src/applications/vaos/services/appointment/transformers.js

```diff
@@ -288,6 +288,62 @@ function setVideoData(appt) {
   };
 }
 
+function getCommunityCareData(appt) {
+  if (!isCommunityCare(appt)) {
+    return {};
+  }
+
+  const apptType = getAppointmentType(appt);
+  return {
+    communityCareProvider:
+      apptType === APPOINTMENT_TYPES.ccAppointment
+        ? {
+            firstName: appt.name?.firstName,
+            lastName: appt.name?.lastName,
+            providerName: appt.name
+              ? `${appt.name.firstName || ''} ${appt.name.lastName || ''}`
+              : null,
+            practiceName: appt.providerPractice,
+            address: appt.address
+              ? {
+                  line: [appt.address.street],
```

Comment:
Just curious as to why we put the street string in an array here. It looked like most references to `address.line` only used the first value.

Reply:
It's following the FHIR Address structure: http://hl7.org/fhir/dstu2/datatypes.html#Address

All the other addresses that come through this service layer are in the FHIR Address format, and we do get some benefits from a consistent address format throughout the app, so I kept that, even though these provider objects are not FHIR compliant overall.